docs/guide/http_cache.md

```diff
@@ -390,84 +390,67 @@ ezpublish:
     php app/console cache:clear -vv --env=prod;
     ```
 
-    #### Configuration on platform.sh
+    #### Configuration on Platform.sh
 
     #### Configure the Fastly extension
 
-    If you want to use Fastly on platform.sh it is recommended to set the setting using the environment variables because
+    If you want to use Fastly on Platform.sh it is recommended to set the setting using the environment variables because
     you then don't need to store service ID and keys in git.
 
-    In order to create new variables on an environment, select your environment (for instance "Master") in the platform.sh
-    dashboard. Then click "Configure environment" and "VARIABLES". To add a variable, click the "Add variable" button.
-    When providing the name for the variable, remember to prefix the name with `env:`", or else platform.sh will only add the
-    settings as part of the $PLATFORM_VARIABLE. Do **not** enable the "JSON value" checkbox.
-
-    Example:
-
+    The easiest way to set environment variables is using the [Plaform.sh CLI](https://docs.platform.sh/gettingstarted/cli.html).
+    Because this is an Enterprise project, you can set the values on the `production` branch specifically
+    and they will be replicated to the production cluster.  Optionally you can set a different set of 
+    credentials on the `staging` branch to test Fastly on the staging instance.
+    
+    Execute the following commands from your local git checkout:
+    
     ```
-    Name: env:HTTPCACHE_PURGE_TYPE
-    Value: fastly
+    platform variable:set -e production env:HTTPCACHE_PURGE_TYPE fastly
+    platform variable:set -e production env:FASTLY_SERVICE_ID YOUR_ID_HERE
+    platform variable:set -e production env:FASTLY_KEY YOUR_ID_HERE
     ```
 
-    Repeat the procedure to set all environment variables mentioned earlier in this document.
-
-    You may export the variables as project variables instead of environment variables. Besides clicking "Configure Project"
-    instead of "Click environment" you follow the same procedure. Ensure that the "Visible at runtime" checkbox is enabled.
-
     ### Configure DNS and hostnames on Fastly
 
     The domain name of your site (for instance `www.foobar.com`) must point to the Fastly CDN.
     See the [Fastly documentation](https://docs.fastly.com/guides/basic-setup/adding-cname-records)
     for details on how to do that.
 
     Next, Fastly needs to know the hostname of your real webserver at platform.sh.
-    You may use the hostname already generated by platform.sh (for instance `master-7abcabc-qwertyqwerty.eu.platform.sh`).
+    You may use the hostname already generated by Platform.sh (for instance `master-7rqtwti-qwertyqwerty.eu.platform.sh`).
```

Comment:
Since this is a enterprise project, won't it typically be:
'foobar.com.c.companyname.ent.platform.sh' ?